File: src/main/java/db/stats/CreepsPerMinDAO.java
package db.stats;

import model.stats.CreepsPerMin;

/**
 * @author Josiah Kendall
 *
 * The Data access object for creeps per min.
 */
public interface CreepsPerMinDAO {

    int saveCreepsPerMin(CreepsPerMin creepsPerMin);
    void deleteCreepsPerMin(long creepsPerMinId);
    CreepsPerMin getCreepsPerMin(long creepsPerMinId);

}

File: processing/src/main/java/db/stats/CreepsPerMinDAO.kt
package db.stats

import model.stats.CreepsPerMin

/**
 * @author Josiah Kendall
 * *
 * * The Data access object for creeps per min.
 */
interface CreepsPerMinDAO {

    fun saveCreepsPerMin(creepsPerMin: CreepsPerMin): Long
    fun deleteCreepsPerMin(creepsPerMinId: Long)
    fun getCreepsPerMin(creepsPerMinId: Long): CreepsPerMin
}
